iam-firewall-ips/iam-firewall-ips-mad.txt

```diff
@@ -0,0 +1,24 @@
+13.120.65.107
```

Comment:
It looks like this is the first time we have added IPs to these lists that don't have a /XX at the end, why is that?

Reply:
@bradbehle I don't know the history here, but all of the existing regions have the 3 ms tugboat istio-proxy addrs listed w/o the suffix as well. e.g. for us-south: [dal10](https://github.com/IBM-Cloud/kube-samples/blob/4b62c95c3bed0733e3852749d8ad18770460263c/iam-firewall-ips/iam-firewall-ips-dal.txt#L116), [dal12](https://github.com/IBM-Cloud/kube-samples/blob/4b62c95c3bed0733e3852749d8ad18770460263c/iam-firewall-ips/iam-firewall-ips-dal.txt#L28), [dal13](https://github.com/IBM-Cloud/kube-samples/blob/4b62c95c3bed0733e3852749d8ad18770460263c/iam-firewall-ips/iam-firewall-ips-dal.txt#L166). 

That said, I certainly wouldn't mind seeing `/32` appended to all of them in all the regions.

Reply:
Hmm, I missed that initially.  Since whoever uses these lists doesn't seem to mine a few of the entries not having a suffix, I'm fine with them as is.